### PR TITLE
Job lisiting active btn fix

### PR DESCRIPTION
**Pull request by rhlc**

### CODE REVIEW

File: src/components/Jobs/JobDetails/index.tsx [hunk 1]
```diff
@@ -350,8 +350,15 @@ const JobDetailsEdit: FC<IJobDetailsEdit> = ({
     setEditorVal(jobEntryData.description);
   }, []);
 
+  const getStatus = () => {
+    if (jobEntryData.jobStatus === 'inactive') {
+      return false;
+    }
+    return true;
```
Comment (GermaVinsmoke) [inline src/components/Jobs/JobDetails/index.tsx]: `return jobEntryData.jobStatus === 'active'`
A simpler way to write the condition

File: src/components/Jobs/JobDetails/index.tsx [hunk 2]
```diff
@@ -426,13 +434,12 @@ const JobDetailsEdit: FC<IJobDetailsEdit> = ({
...
     setShowView(false);
   };
-
   return (
     <>
       <span className={styles['inline-job-title']}>
         <div className={styles['inline-job-div']}>
           <input
-            placeholder="Job Title here"
+            placeholder={jobEntryData.title ?? jobListingData.title}
```
Comment (GermaVinsmoke) [inline src/components/Jobs/JobDetails/index.tsx:437]: What's the diff between jobEntryData and jobListingData

Comment (rhlc) [reply src/components/Jobs/JobDetails/index.tsx:437]: jobEntryData is old task data and on editing it becomes jobListingData

Comment (GermaVinsmoke) [reply src/components/Jobs/JobDetails/index.tsx:437]: okay

**Review by GermaVinsmoke — APPROVED** [review]

**Pull request merged**